doc_src/cmds/complete.rst

```diff
@@ -76,7 +76,7 @@ Command specific tab-completions in ``fish`` are based on the notion of options
 
 - Short options, like ``-a``. Short options are a single character long, are preceded by a single hyphen and can be grouped together (like ``-la``, which is equivalent to ``-l -a``). Option arguments may be specified by appending the option with the value (``-w32``), or, if ``--require-parameter`` is given, in the following parameter (``-w 32``).
 
-- Old-style long options, like ``-Wall`` or ``-name``. Old-style long options can be more than one character long, are preceded by a single hyphen and may not be grouped together. Option arguments are specified in the following parameter (``-ao null``) or after a ``=`` (``-ao=null``).
+- Old-style options, long like ``-Wall`` or ``-name`` or even short like ``-a``. Old-style options can be more than one character long, are preceded by a single hyphen and may not be grouped together. Option arguments are specified by default following a space (``-foo null``) or after ``=`` (``-foo=null``).
```

Comment:
I was a bit thrown off by the "by default", but it turns out old-style options really don't need "--require-parameter" to accept an option in the next parameter?

That might actually be worth a change!

(not that "require-parameter" is the right way around - it should have been "--optional-parameter" for the few times you have an optional parameter)